static/app/views/issueDetails/streamline/sidebar/activityDrawer.tsx

```diff
@@ -39,10 +65,28 @@ export function ActivityDrawer({group, project}: ActivityDrawerProps) {
       </EventDrawerHeader>
       <EventNavigator>
         <Header>{t('Activity')}</Header>
+        <FilterButton
```

Comment:
If this was already decided by Vu then disregard, but IMO this should look more like the filters in breadcrumbs. To me the chat icon isn't as intuitive and is easy to miss

![CleanShot 2025-02-19 at 16 06 56](https://github.com/user-attachments/assets/6876bc6d-d70a-4ed9-9eb8-4e0391821198)

Reply:
talked to vu and updated the design